Sources/WalletConnectKMS/Keychain/GroupKeychainStorage.swift

```diff
@@ -0,0 +1,104 @@
+
+import Foundation
+
+public final class GroupKeychainStorage: KeychainStorageProtocol {
```

Comment:
a lot of code duplicates with KeychainStorage ig, is it possible to reuse one instance? And configure it as shared/not shared

Reply:
I know and I was thinking about it but didn't want to modify existing code. 
will take a look again, any suggestions?